plugins/by-name/hunk/default.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  helpers,
+  config,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "hunk";
+  packPathName = "hunk.nvim";
+  package = "hunk-nvim";
+
+  description = ''
+    A tool for splitting diffs in Neovim
+
+     Either `plugins.web-devicons` or `plugins.mini`* must be enabled to have icons in the file tree.
+
+     *If using `plugins.mini`, you must enable the `icons` module.
```

Need to escape the `*`, as markdown uses them for italics and bullet points.

```suggestion
     Either `plugins.web-devicons` or `plugins.mini`\* must be enabled to have icons in the file tree.

     \*If using `plugins.mini`, you must enable the `icons` module.
```

---

Also, "must" feels too strong for an optional dependency. I'd suggest something like:

> If you wish to display icons in the file tree, you should enable either `plugins.web-devicons` or `plugins.mini`. If using `plugins.mini`, you must enable the `icons` module.

?

---

Reworded, so no escapes needed, fixed!